file.go

```diff
@@ -578,9 +511,8 @@ func (f *File) Undo(isundo bool) (q0, q1 int, ok bool) {
 			f.mod = u.mod
 			f.treatasclean = false
 			f.b.Delete(u.p0, u.p0+u.n)
-			for _, text := range f.text {
-				text.deleted(u.p0, u.p0+u.n)
-			}
+			//TODO(sn0w): Find a home for this:
```

Comment:
This breaks `Undo`. You need to plumb this. `File` needs to replay the changes to `ObservableEditableBuffer` and it needs to broadcast them to the observers.